dev-lisp/c2ffi/c2ffi-14.0.0.20220729-r1.ebuild

```diff
@@ -19,3 +19,5 @@ KEYWORDS="~amd64"
 DEPEND="sys-devel/clang:14
```

Comment:
Move `LLVM_MAX_SLOT` to before the dependencies and don't hardcode the slot?

Reply:
Why not hardcode the slot? It needs 14, won't compile otherwise.

Reply:
Because `LLVM_MAX_SLOT` is already there for a similar purpose.